.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+.idea
```

Comment:
It's not necessary.
if u want to disable some files from the stage u must to create user specific `.gitignore` file and added it to ur git config. 

For a sample like this:
```
# ~/.gitconfig

[core]
	excludesfile = /home/user/.gitignore
```
```
# /home/user/.gitignore
/.idea/
```